### PR TITLE
Fixing path changes, case issues, and missing berlin data

### PR DESCRIPTION
**Pull request by sjsrey**

### CODE REVIEW

**Comment by sjsrey** [comment]
Note that some of the links at natural earth evidently have moved. Also a number of the attributes have had their case changed.

**Comment by jorisvandenbossche** [comment]
Can you do a "Kernel ->Restart & Clear Output" on the notebooks 1-4? Because now the diff is a bit difficult to see

**Comment by sjsrey** [comment]
Closing as needs fixing

**Comment by sjsrey** [comment]
will do on the "Kernel ->Restart & Clear Output" on the notebooks 1-4?"

**Comment by sjsrey** [comment]
Notebooks are fine, the only real changes that have to made for things to run on my end are:

https://github.com/geopandas/scipy2018-geospatial-data/pull/4/commits/8752c926051a8ebd938efe4d91f53fd3f84bb521

and

https://github.com/geopandas/scipy2018-geospatial-data/pull/4/commits/c679b4e307f66dd64fadf10d3404adca32570757#diff-b7cb75ca59993a879e29bed8cd8469b8

**Comment by jorisvandenbossche** [comment]
There is already a berlin-neighbourhoods.geojson, the shapefile you added is something different?

**Comment by sjsrey** [comment]
Notebook 06 uses the shp file.

**Comment by ljwolf** [comment]
Let's just update the geojson with the `median_price` field and drop the two disconnected polys. We're not running the weights notebook, and I don't think it makes sense to build the shp then. 

I can do this right now.

**Comment by ljwolf** [comment]
I've added my changes to @sjsrey, but my matplotlib rc is nonstandard, so coloring on some plots looks different.

**Comment by ljwolf** [comment]
OK, plots should look like standard mpl. sorry about that.

File: README.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ distribution, found at https://www.anaconda.com/download/).
 Once this is installed, the following command will install all required packages in your Python environment:
 
 ```
-conda env create -f environment.yml
```
Comment (ljwolf) [inline README.md]: I thought about this a bit... We're teaching at a special conference, so maybe the user might be making a bunch of different conda environments and this makes sense. But, when we do one-offs, we might want to just dump it into root, cause we want to just get to a working environment. 

Also, if we do want a name, I think it should be put it in the `environment.yml` as

```
- name: scipygeo2018
```

**Comment by ljwolf** [comment]
If you'd like, @sjsrey, `git pull --rebase` your `sjr-add-gds-content` branch and rerun `06,07,08` if you'd like to confirm the swap to a fixed & clean `berlin-neighbourhoods.geojson` has worked.

File: 01-introduction-geospatial-data.ipynb [hunk 2]
```diff
@@ -36,7 +36,7 @@
     "\n",
     "We can use the GeoPandas library to read many of those GIS file formats (relying on the `fiona` library under the hood, which is an interface to GDAL/OGR), using the `geopandas.read_file` function.\n",
     "\n",
-    "For example, let's start by reading a shapefile with all the countries of the world (from http://www.naturalearthdata.com/downloads/110m-cultural-vectors/110m-admin-0-countries/), and inspect the data:"
+    "For example, let's start by reading a shapefile with all the countries of the world (from https://www.naturalearthdata.com/http//www.naturalearthdata.com/download/110m/cultural/ne_110m_admin_0_countries.zip), and inspect the data:"
```
Comment (jorisvandenbossche) [inline 01-introduction-geospatial-data.ipynb]: I changed the wording here a little bit, as the participants are not expected to download it themselves, the data will be provided in the `/data` directory (I added them earlier today)

**Pull request merged**